pom.xml

```diff
@@ -82,7 +82,7 @@
     <version.junit-jupiter>5.8.1</version.junit-jupiter>
     <!-- used for deployment to SonarSource Artifactory -->
     <gitRepositoryName>sonar-analyzer-commons</gitRepositoryName>
-    <maven.compiler.release>8</maven.compiler.release>
+    <maven.compiler.release>11</maven.compiler.release>
```

Comment:
⚠️  This change is not required and will impact: sonar-css, sonar-flex, sonar-rpg, sonar-xml, sonar-pli, sonar-vb, sonar-tsql, sonar-plsql, sonar-abap, sonar-cobol
Why?